api/envoy/extensions/quic/server_preferred_address/v3/empty_server_preferred_address_config.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+package envoy.extensions.quic.server_preferred_address.v3;
+
+import "udpa/annotations/status.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.quic.server_preferred_address.v3";
+option java_outer_classname = "EmptyServerPreferredAddressConfigProto";
+option java_multiple_files = true;
+option go_package = "github.com/envoyproxy/go-control-plane/envoy/extensions/quic/server_preferred_address/v3;server_preferred_addressv3";
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: QUIC server preferred address config]
+// [#extension: envoy.quic.server_preferred_address.empty]
+
+// Configuration for a server's preferred address config implementation which doesn't provide any preferred addresses.
+message EmptyServerPreferredAddressConfig {
```

Comment:
It seems like this matches the default behavior. is that right?

Reply:
That's my initial approach, but a new extension category without an extension implementation failed one of the rst document generation step and I had to add an extension referencing the new category in source/extensions/extensions_metadata.yaml.

Reply:
That being said, if I convert the testing extension into non-testing one, the doc generation won't complain any more. And I should be able to leave the extension unset if not configured. Let me try this first.